assets/queries/ansible/aws/IAM_policies_attached_to_user/query.rego

```diff
@@ -0,0 +1,27 @@
+package Cx
+
+CxPolicy [  result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  awsApiGateway := task["community.aws.iam_managed_policy"]
+  contains(awsApiGateway.state, "present")
+  type := awsApiGateway.iam_type
+  contains(type,"user")
+  clusterName := task.name
+  result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.iam_managed_policy}}.iam_type", [clusterName]),
+                "issueType":        "RedundantAttribute",
```

Comment:
Although the Terraform's version for this query has "RedundantAttribute" as issue type, I think, for Ansible, should be "IncorrectValue" because the "iam_type" field is required for this task.